internal/renderers/femtovg/Cargo.toml

```diff
@@ -33,7 +33,7 @@ once_cell = "1.5"
 pin-weak = "1"
 scoped-tls-hkt = "0.1"
 raw-window-handle = { version = "0.5", features = ["alloc"] }
-femtovg = { version = "0.7.0" }
+femtovg = { version = "0.7.0", default-features = false }
```

Comment:
This doesn't seem quite right to me. I believe the code here needs the image-loading feature of femtovg.

Reply:
Why do we need that? The image are loaded by i-slint-core.
if we were using this feature, i think we'd have a compilation error.

Reply:
Yes, for our images that's true. But the image dependency in femtovg also enables it to load pngs from ttf files, for color emojis:

https://github.com/femtovg/femtovg/blob/46fdb65a569d75639f67d995b11e6fb7ea30003a/src/text/font.rs#L195

Reply:
Ah ok, i didn't know that.
I removed this part then.
Thanks for noticing.